packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -98,9 +101,21 @@ export default class Photo extends PureComponent {
     )
   }
 
+  get isFallback() {
```

Comment:
I believe this would be the same:
```suggestion
  get isFallback() {
    return this.img.current && (this.img.current.src === this.props.fallbackUrl)
  }
```

Reply:
👍